### ISSUE TITLE
bash: /dev/fd/63: No such file or directory

### ISSUE THREAD

**Issue by WoutB** [lukasjapan/bt-speaker#53]
Help, I can't install

**Comment by ritiek** [comment]
I had a similar problem. Downloading the install script locally and then executing it worked for me.
```bash
$ wget https://raw.githubusercontent.com/lukasjapan/bt-speaker/master/install.sh
$ bash install.sh
```

**Comment by WoutB** [comment]
Thanks, this did the trick. But it still won't play any music.. I can connect, but I have no sound

**Comment by romankc** [comment]
WoutB, have you been able to get it to work? I am having the same problem.

**Comment by WoutB** [comment]
Nope, gave it up. But I have another solution: I'm going to use Kodi (libre elec) -> search it. It's a google chromecast/smart tv thing where you can watch movies from a homecloud.. BUT you can install plugins (i've installed youtube & spotify). It's with a super easy interface. After installing the spotify plugin I was able to select this Pi in the spotify apps (the available device button). I can also select share -> play on Kodi on every youtube video. 

Just download & install Kodi, set up the network acces, search the plugins you need, install them. Next search the settings so you can choose aux as an audio output. Disconnect the screen, plugin the aux , and enjoy! This was for me the best solution for an audiocast alternative (on a rasp pi zero w)
